yarn_api_client/hadoop_conf.py

```diff
@@ -41,19 +37,18 @@ def check_is_active_rm(rm_web_host, rm_web_port):
     else:
         if response.getheader('Refresh', None) is not None:
             return False
-        return True
+    return True
 
 
-def get_resource_manager_host_port():
+def get_resource_manager_host_port(timeout=30):
```

Comment:
Seems like this should be renamed to `get_resource_manager_endpoint`

Reply:
Adjusted